src/media/css/lib/mixins.styl

```diff
@@ -186,8 +186,24 @@ category-icon-position($slug, $scale=1) {
     background-position: ($positions[0] * $cat-icon-size-default * $scale * -1) ($positions[1] * $cat-icon-size-default * $scale * -1);
 }
 
+ltr() {
+    .html-ltr & {
```

Comment:
maybe might as well do it now as we're adding the mixins. 

performance changes might be negligible since if we change it, we can remove the code that sets html-{dir}.